src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2319,6 +2319,24 @@ because xbuild doesn't support framework reference assemblies.
 	ResourceDirectories="$(MonoAndroidResDirIntermediate);@(LibraryResourceDirectories)"
 	ResourceNameCaseMap="$(_AndroidResourceNameCaseMap)"
   />
+  <Delete Files="@(_LibraryResourceHashDirectories->'$(_AndroidLibraryFlatArchivesDirectory)\%(Hash).stamp');$(_AndroidLibraryFlatArchivesDirectory)\compiled.stamp"
+	Condition=" '$(AndroidUseAapt2)' == 'True' "
+  />
+  <Aapt2Compile
```

Comment:
Will the extra `<Aapt2Compile/>` call be skipped if there are no custom views? Is it worth worrying about that?

I'm mainly wondering in the case of a Xamarin.Forms app using aapt2, they might not have any custom views.

Reply:
I think I need to add some checks to make sure we don't run it.